### ISSUE TITLE
code corresponding to Eq.4 in paper

### ISSUE THREAD

**Issue by CIntellifusion** [roboterax/video-prediction-policy#2]
Thanks for you excellent work! 
I am wondering where and how is the balance between different dataset types during the stage 1 svd finetuning? 
I notice that in stage 1,  different dataset are mixed, but no explicit lambda or loss weight in training loop. 

Do I miss something? 

And another question is all the three terms in eq4 are simple diffusion loss, right?

**Comment by Robert-gyj** [comment]
Thank you for your interest in our work! 

Actually we directly sample different number of items from different datasets in **each batch**, the sample ratio can be explicitly defined in https://github.com/roboterax/video-prediction-policy/blob/main/video_conf/train_calvin_svd.yaml#L25 
The overall effect is equivalent to the equation 4 of the paper. Specifically,  start from the line 239 of the **getitem** function in dataset, we first select the sampled datasets, then we sample the specific item:
https://github.com/roboterax/video-prediction-policy/blob/main/video_dataset/dataset_mix.py#L239

Yes, three terms are just diffusion loss.

**Comment by CIntellifusion** [comment]
Got it. Thanks for your help!
